main.rb

```diff
@@ -0,0 +1,92 @@
+require_relative "store"
+require 'terminal-table/import'
+
+class BillingMachine 
+  attr_reader :total, :menu, :order, :amount_saved, :reciept_table, :item, :quantity, :price, :sale_qun, :sale_price, :items_price
+
+  def initialize(order)
+    @total = 0
+    @menu = STORE::MENU
+    @order = order
+    @amount_saved = 0
+    @reciept_table = []
+    @item = nil,
```

Comment:
item, price, sale_price, sale_qun are not attributes of BillingMachine. They are attributes of a menu item for which you're calculating cost. Hence, don't use these as instance variables of BillingMachine.